src/divider.jsx

```diff
@@ -51,7 +42,6 @@ const Divider = ({inset, style, ...other}, {muiTheme = ThemeManager.getMuiTheme(
 
 Divider.propTypes = propTypes;
 Divider.defaultProps = defaultProps;
-Divider.contextTypes = contextTypes;
-Divider.childContextTypes = childContextTypes;
+Divider = muiThemeable(Divider);
 
 export default Divider;
```

Comment:
My mistake, not an eslint error, there is a runtime error saying `Uncaught Error: No suitable component definition found.`

Will have to investigate this as well.